src/containers/Manifest.js

```diff
@@ -37,7 +37,7 @@ const lifecycleMethods = {
     propsVerification(this.props)
     if (this.props.inMemoryData) {
       this.props.setInMemoryData(this.props.name, this.props.inMemoryData)
-    } else {
+    } else if (this.props.autoLoad !== false) {
```

Comment:
why can't you do `this.props.autoLoad === true`?  isn't autoLoad in the initValues of the reducer?